homeassistant/components/tradfri.py

```diff
@@ -23,8 +24,7 @@
                 '#aiocoap==0.3']
 
 DOMAIN = 'tradfri'
-GATEWAY_IDENTITY = 'homeassistant'
-CONFIG_FILE = '.tradfri_psk.conf'
+CONFIG_FILE = '.tradfri_identity.conf'
```

Comment:
File change is fine if we manage to merge this before 57. After that it will be an unnecessary breaking change